### PR TITLE
OCPVE-588: chore: refactor AWS Disk Setup to use correct logs/ctx

### PR DESCRIPTION
**Pull request by jakobmoellerdev**
This PR upgrades The AWS sdk, refactors the AWS client calls to all use a Context from Before/After Suite in Ginkgo, and passes a structured `logr.Logger` from Ginkgo that is `-v` aware instead of using `fmt.Println` in the disk setup utility for our AWS scenarios

### CODE REVIEW

**Comment by jakobmoellerdev** [comment]
/hold requires https://github.com/openshift/release/pull/42554

can already be reviewed though.

File: test/e2e/disk_setup.go [hunk 1]
```diff
@@ -20,73 +20,82 @@ import (
...
-
 	// get nodes
+	By(fmt.Sprintf("getting all worker nodes by label %s", labelNodeRoleWorker))
 	nodeList := &corev1.NodeList{}
-	err := crClient.List(ctx, nodeList, client.HasLabels{labelNodeRoleWorker})
-	if err != nil {
-		fmt.Printf("failed to list nodes\n")
-		return err
+	if err := crClient.List(ctx, nodeList, client.HasLabels{labelNodeRoleWorker}); err != nil {
+		return fmt.Errorf("could not list worker nodes nodes for Disk setup: %w", err)
```
Comment (suleymanakbas91) [inline test/e2e/disk_setup.go:35]:
```suggestion
		return fmt.Errorf("could not list worker nodes for Disk setup: %w", err)
```

**Review by suleymanakbas91 — COMMENTED** [review]
Great work!
/lgtm
/approve

**Pull request merged**